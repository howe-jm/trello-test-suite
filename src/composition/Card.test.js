import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';

// This is the test case.
it('Renders without crashing', () => {
  // Create an element to test the component in.
  const div = document.createElement('div');
  // Render the component to that element.
  ReactDOM.render(<Card />, div);
  // Clean up the created element.
  ReactDOM.unmountComponentAtNode(div);
});
