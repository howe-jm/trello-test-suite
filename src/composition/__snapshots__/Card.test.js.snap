// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
